README.md

```diff
@@ -274,15 +274,23 @@ The STH component server is ready to be started.
 1. To run the STH server, just execute:
 <pre> npm start </pre>
 
+The STH component provides the user with 2 mechanisms to configure the component to the concrete needs of the user:
+
+* Environment variables, which can be set assigning values to them or using the `sth_default.conf` file if a packaged
+version of the STH component is used.
+* The [`config.js`](https://github.com/telefonicaid/IoT-STH/blob/develop/config.js) located at the root of the STH component code, a JSON formatted file including the configuration properties.
+
+It is important to note that environment variables, if set, take precedence over the properties defined in the `config.js` file.
+
 The script accepts the following parameters as environment variables:
 
 - STH_HOST: The host where the STH server will be started. Optional. Default value: "localhost".
-- STH_PORT: The port where the STH server will be listening. Optional. Default value: 8666.
+- STH_PORT: The port where the STH server will be listening. Optional. Default value: "8666".
```

Comment:
Issue https://github.com/telefonicaid/IoT-STH/issues/82 created to track this ;) Thanks @frbattid !